Mobile/NUI_CustomLayout/NUI_CustomLayout/ItemLayout.cs

```diff
@@ -0,0 +1,122 @@
+/*
+ * The Item Layout has two modes depending on the number of text labels inserted into it. 
+ * Structure is depicted below. 
+ * 
+ *   +------------------------------------------------------------------------+       
+ *   |  +------+   +--------------------------------------------------------+ |       
+ *   |  |      |   |                                                        | |       
+ *   |  | ICON |   |   TITLE                                                | |       
+ *   |  |      |   |                                                        | |       
+ *   |  +------+   +--------------------------------------------------------+ |       
+ *   +------------------------------------------------------------------------+       
+ *                                                                                      
+ *   +------------------------------------------------------------------------------+       
+ *   |              +-------------------------------------------------------------+ |       
+ *   |  +-------+   | TITLE                                                       | |       
+ *   |  |       |   |                                                             | |       
+ *   |  | ICON  |   +-------------------------------------------------------------+ |       
+ *   |  |       |   +-------------------------------------------------------------+ |       
+ *   |  +-------+   | DESCRIPTION                                                 | |       
+ *   |              |                                                             | |       
+ *   |              +-------------------------------------------------------------+ |       
+ *   +------------------------------------------------------------------------------+      
+*/
+
+namespace SimpleLayout
+{
+    internal class ItemLayout : LayoutGroup
+    {
+        private static readonly int MARGIN = 5;
+
+        private static readonly int TEXT_LEFT_MARGIN = 120;
+        private static readonly int TEXT_RIGTH_MARGIN = 710;
+        private static readonly int DESCRIPTION_TOP_MARGIN = 70;
+        private static readonly int TEXT_HEIGHT = 50;
+
+        protected override void OnMeasure(MeasureSpecification widthMeasureSpec, MeasureSpecification heightMeasureSpec)
+        {
+            var itemWidth = new LayoutLength(0);
+            var itemHeight = new LayoutLength(0);
+            var iconHeight = new LayoutLength(0);
+
+            float labelMaxWidth = 0;
+
+            foreach (LayoutItem childLayout in LayoutChildren)
+            {
+                if (childLayout != null)
+                {
+                    Type imageType = typeof(Tizen.NUI.BaseComponents.ImageView);
+                    Type textType = typeof(Tizen.NUI.BaseComponents.TextLabel);
+                    Type childType = childLayout.Owner.GetType();
+
+                    MeasureChild(childLayout, widthMeasureSpec, heightMeasureSpec);
+
+                    if (childType.Equals(textType))
```

Comment:
What do you think of using `is` keyword?
You can simplify pattern matching code.
```suggestion
                    if (childLayout.Owner is TextLabel)
```

Reply:
Fixed